### DISCUSSION TITLE
Tutorial on how to use theme.json filters

### DISCUSSION THREAD

**Discussion by ndiego** [WordPress/developer-blog-content#82] [Topic Ideas]
[theme.json filters](https://make.wordpress.org/core/2022/10/10/filters-for-theme-json-data/) were introduced in WordPress 6.1, and allow developers to modify theme.json settings and styles in unprecedented ways. Unfortunately, there is little documentation and few real-world examples. Educating the community, especially agency developers, on these new editor curation techniques would hopefully increase block adoption among an important demographic.

This post should cover the following:

- Illustrate the need for more editor curation tools for advanced builders.
- Provide an overview theme.json filters and how they help address this need
- Provide 3-4 real-world examples

**Comment by bph** [comment]
Converted to an issue. Continue discussion in the comments there: 

- #86

**Comment by ndiego** [comment]
The post has been published here: [How to modify theme.json data using server-side filters](https://developer.wordpress.org/news/2023/07/how-to-modify-theme-json-data-using-server-side-filters/)

Thanks to everyone who helped out!
